client/src/components/registration.js

```diff
@@ -0,0 +1,48 @@
+import { Component } from 'preact'
+import withStyles from './jss'
+
+export default withStyles({
+  root: {
+    padding: '25px'
+  },
+  submit: {
+    'margin-top': '25px'
+  }
+}, class Register extends Component {
+  componentDidMount () {
+    document.title = 'Registration'
+  }
+
+  render ({ classes }) {
+    return (
+      <div class='row u-center'>
+        <div class={classes.root + ' col-6'}>
+          <div class='form-section'>
+            <div class='input-control'>
+              <input class='input-contains-icon' id='name' name='name' placeholder='Team Name' type='text' value='' />
+              <span class='icon'>
+                <i class='fa fa-wrapper fa-user-circle small' />
+              </span>
+            </div>
+            <div class='input-control'>
+              <input class='input-contains-icon' id='email' name='email' placeholder='Email' type='text' value='' />
```

Comment:
Input box should probably be their own components.

Reply:
Will refactor in separate PR